server.js

```diff
@@ -69,11 +82,14 @@ const submitSearch = (buffer) =>
       return;
     }
     if (!searchResult.docs) {
-      resolve({ text: "Backend server error, please try again later." });
+      resolve({
+        text:
+          "Error with search response. Is your query correctly formatted? Note: URL searches must be for static images, not gifs",
```

Comment:
I know the original error message is a bit vague. But this is even more confusing if it shows messages about image URLs to all users (even they they are not searching by URL).